src/app/content/search/hooks/index.ts

```diff
@@ -84,9 +85,11 @@ export const receiveSearchHook: ActionHookBody<typeof receiveSearchResults> = (s
   };
 
   const action = stripIdVersion(page.id) === stripIdVersion(targetPage.id) ? replace : push;
+  const contentStyle = navigationQuery['content-style'] as string | undefined;
   const search = queryString.stringify({
+    'content-style': contentStyle,
```

Comment:
i feel like you want a centralized config somewhere for like... 'persistentQueryParameters` that includes the search query and content style, and merge that under the given nav options in the router dispatch, rather than having to update all these different places